modules/Bio/EnsEMBL/Variation/Pipeline/ProteinFunction/InitJobs.pm

```diff
@@ -228,14 +228,11 @@ sub fetch_input {
     my $var_dbc = $self->get_species_adaptor($group)->dbc;
     unless (defined $old_server_uri) {
         my $user     = $var_dbc->user;
+        my $pass     = $var_dbc->pass;
         my $port     = $var_dbc->port;
         my $host     = $var_dbc->host;
-        my $species  = $self->param('species');
         my $release  = $self->param('ensembl_release') - 1;
-        my $assembly = $self->param('assembly');
-        $old_server_uri ||= sprintf("mysql://%s@%s:%s/%s_%s_%s_%s",
-                                    $user, $host, $port,
-                                    $species, $group, $release, $assembly);
+        $old_server_uri ||= sprintf("mysql://%s:%s@%s:%s/%s", $user, $pass, $host, $port, $release);
```

Comment:
We don't need the password. 
For example, this works: `mysql://ensro@mysql-ens-mirror-1:4240/109`

Reply:
It will work if we explicitly give it as cli param. Otherwise when it is formed here in the `InitJobs` the params are coming from the registry file and the user here is what we have for variation server in the registry - which is always the user with write-permission for this pipeline. Hence the password is needed.